web-console/Storage/OCS/OCS _Installation_Workflow.md

```diff
@@ -0,0 +1,83 @@
+# OCS Install Workflow
+
+### OperatorHub
+
+There will be 2 OCS operator entries in OperatorHub covering each of the following modes. 
+* Converged Mode -- Local Cluster / Hyperconverged 
+* Independent Mode -- External Cluster (Ceph Cluster is deployed externally)
+
+### Converged Mode Design
+
+OperatorHub
+![OperatorHub operators view](img/OCS-operator-view.png)
+
+Installation
+![OperatorHub operators view](img/OCS-Install-step1.png)
+
+![OperatorHub operators view](img/OCS-Install-step2.png)
+
+After the subscription process, the user will see the subscribed operator page. 
+
+![subscription operators page](img/OCS_Subscription_page.png) 
+
+In the installed operator view the user can see the OCS operator installed. 
+
+![OperatorHub operators view](img/Installed_operators_list.png)
+
+
+# The OCS installation process
+OCS Overview page
+![OCS-View](img/OCS-View.png)
```

Comment:
This page layout has changed for 4.2 to display basic overview information then the CSV overview below. Subscription page should also be a tab to the right of YAML.